src/Sentry.Unity.Editor/AutoInstrumentation/SentryAssemblyResolver.cs

```diff
@@ -0,0 +1,40 @@
+using System.IO;
+using Mono.Cecil;
+
+namespace Sentry.Unity.Editor
+{
+    public class SentryAssemblyResolver : BaseAssemblyResolver
+    {
+        private readonly string _workingDirectory;
+        private readonly DefaultAssemblyResolver _defaultResolver;
+
+        public SentryAssemblyResolver(string workingDirectory)
+        {
+            _workingDirectory = workingDirectory;
+            _defaultResolver = new DefaultAssemblyResolver();
+        }
```

Comment:
Nothing wrong here, but this is a good place to use a field initializer with targeted-type-new.

```suggestion
        private readonly string _workingDirectory;
        private readonly DefaultAssemblyResolver _defaultResolver = new();

        public SentryAssemblyResolver(string workingDirectory)
        {
            _workingDirectory = workingDirectory;
        }
```